<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>604</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wizard Studio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sources.qrc">
    <normaloff>:/sources/img/logo.svg</normaloff>:/sources/img/logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitterAllContent">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitterStackedWidgetAndSidebar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="opaqueResize">
        <bool>true</bool>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="StartSection" name="startSection"/>
        <widget class="WorkSection" name="workSection"/>
       </widget>
       <widget class="SideBar" name="sideBar">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="indentation">
         <number>15</number>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="Output" name="output" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>180</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionNewFile"/>
    <addaction name="actionNewProject"/>
    <addaction name="separator"/>
    <addaction name="actionSaveFile"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionOutputConsole"/>
    <addaction name="actionSetting"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionCompile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuBuild"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/sources/img/icons/createFile.svg</normaloff>:/sources/img/icons/createFile.svg</iconset>
   </property>
   <property name="text">
    <string>New file</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/sources/img/icons/openFile.svg</normaloff>:/sources/img/icons/openFile.svg</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionOutputConsole">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/sources/img/icons/Output.svg</normaloff>:/sources/img/icons/Output.svg</iconset>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/sources/img/icons/createProject.svg</normaloff>:/sources/img/icons/createProject.svg</iconset>
   </property>
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/sources/img/icons/openProject.svg</normaloff>:/sources/img/icons/openProject.svg</iconset>
   </property>
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/sources/img/icons/saveFile.svg</normaloff>:/sources/img/icons/saveFile.svg</iconset>
   </property>
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/sources/img/icons/run.svg</normaloff>:/sources/img/icons/run.svg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/sources/img/icons/compile.svg</normaloff>:/sources/img/icons/compile.svg</iconset>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/sources/img/icons/setting.svg</normaloff>:/sources/img/icons/setting.svg</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionSidebar">
   <property name="text">
    <string>Sidebar</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StartSection</class>
   <extends>QWidget</extends>
   <header>startsection.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WorkSection</class>
   <extends>QWidget</extends>
   <header>worksection.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SideBar</class>
   <extends>QTreeWidget</extends>
   <header>sidebar.h</header>
  </customwidget>
  <customwidget>
   <class>Output</class>
   <extends>QWidget</extends>
   <header>output.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
